clients/fides-js/rollup.config.mjs

```diff
@@ -20,6 +22,13 @@ const GZIP_SIZE_WARN_KB = 35; // log a warning if bundle size exceeds this
 const GZIP_SIZE_TCF_ERROR_KB = 85;
 const GZIP_SIZE_TCF_WARN_KB = 75;
 
+let GIT_TAG_VERSION = "0.0.0";
+try {
+  GIT_TAG_VERSION = gitTagVersion().split("-")[0];
```

Comment:
nit: I'd recommend putting a log here like:
```
console.log(`Starting FidesJS build and tagging with current git tag (Fides.version=${GIT_TAG_VERSION})...`)
```

With that in there, you can pre-emptively troubleshoot if this doesn't work as expected in CI by looking at the GHA logs when it runs